packages/core/src/internal/share-latest.ts

```diff
@@ -132,6 +144,24 @@ const shareLatest = <T>(
       subscription!.add(pSubs)
     }))
   }
+  result.sB = (callback) => {
+    const sub = result.subscribe({
+      next: callback,
+      error: callback,
+    })
+    return () => sub.unsubscribe()
+  }
+
+  const cachedCallback: WeakMap<Subscription, () => any> = new WeakMap()
+  result.gVS = (subscription) => {
+    if (!subscription) {
+      return result.gV
+    }
+    if (!cachedCallback.has(subscription)) {
+      cachedCallback.set(subscription, () => result.gV(subscription))
+    }
```

Comment:
I'm now thinking this is creating a hard reference to `subscription`, so it won't get GCed. Will need more rework.